src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -237,7 +237,10 @@ export class HuiGlanceCard extends LitElement implements LovelaceCard {
                 .stateObj=${stateObj}
                 .overrideIcon=${entityConf.icon}
                 .overrideImage=${entityConf.image}
-                stateColor
+                .stateColor=${entityConf.state_color === false ||
+                entityConf.state_color
+                  ? entityConf.state_color
+                  : this._config!.state_color}
```

Comment:
We should add nullish coalescing:
`entityConf.state_color ?? this._config!.state_color`

Reply:
I was not aware of such a thing 👍 nice